### DISCUSSION TITLE
Github Overview Heatmap repsonsive issue.

### DISCUSSION THREAD

**Discussion by 0xdanieltran** [community/community#15826] [Other Feature Feedback, Questions, & Ideas]
Please see heatmap of github overview page in 1024 * 768 screeen.
Now I am using this screen and now it seems like responsive error.
I have attached image.
https://drive.google.com/file/d/1qJ2I14xv-tFATEilulKWhNlS2fcHJaZF/view?usp=sharing
If this is error, hope github support team to fix this asap.
Thx.

**Comment by LosAhuertos** [comment]
Подрочим?

сб, 30 апр. 2022 г., 05:46 BYD ***@***.***>:
